### ISSUE TITLE
CDN/bundling reveal.js in nbconvert

### ISSUE THREAD

**Issue by zscore** [jupyter/nbconvert#653]
Would it be possible to include a CDN reference for reveal.js slides created with `jupyter nbconvert --to slides`? And/or have an option to download a copy of the reveal library and package it up with the presentation?

**Comment by takluyver** [comment]
cc @damianavila 

Answers may be delayed a while because people are at Jupytercon.

**Comment by zscore** [comment]
ooo hope they're whipping up something good for us!

**Comment by damianavila** [comment]
> Would it be possible to include a CDN reference for reveal.js slides created with jupyter nbconvert --to slides?

Do you mean an option pointing to a specific CDN?

> And/or have an option to download a copy of the reveal library and package it up with the presentation?

That could be an option, I think.

**Comment by zscore** [comment]
yeah I'd prefer to just bundle up the javascript myself. Would we just bring the javascript into the repo itself, or would we download them from somewhere online?

**Comment by damianavila** [comment]
I guess we can provide a way/command to download it. There is not a natural place in nbconvert to add a js library.
Another alternative would be to provide reveal.js with the notebook package (that has a natural place to put js stuff) and then use that from nbconvert. I have done some work in the past to include reveal.js i the notebook and I have to resurrect that code...

**Comment by zscore** [comment]
hey sorry for the delay. yeah, that'd be awesome if you could give us a starting place. do you mean putting reveal.js somewhere over here: https://github.com/jupyter/notebook ?

**Comment by koutbo6** [comment]
setting the correct url as the reveal.js prefix did it for me
`jupyter nbconvert --to slides --reveal-prefix https://cdn.jsdelivr.net/npm/reveal.js@3.6.0 notebook.ipynb`
